pkgs/data/misc/geolite-legacy/default.nix

```diff
@@ -1,27 +1,28 @@
-{ lib, stdenv, fetchurl, zstd }:
+{ lib, stdenv, fetchzip, zstd }:
 
 stdenv.mkDerivation rec {
   pname = "geolite-legacy";
-  version = "20230901";
+  version = "20240720";
 
   # We use Arch Linux package as a snapshot, because upstream database is updated in-place.
-  geoip = fetchurl {
+  geoip = fetchzip {
     url = "https://archive.archlinux.org/packages/g/geoip-database/geoip-database-${version}-1-any.pkg.tar.zst";
-    sha256 = "sha256-H6tv0OEf04TvbhbWsm5vwq+lBj4GSyOezd258VOT8yQ=";
+    hash = "sha256-9rPp1Lu6Q4+Cb4N4e/ezHacpLuUwbGQefEPuSrH8O6o=";
+    nativeBuildInputs = [ zstd ];
+    stripRoot = false;
   };
 
-  extra = fetchurl {
+  extra = fetchzip {
     url = "https://archive.archlinux.org/packages/g/geoip-database-extra/geoip-database-extra-${version}-1-any.pkg.tar.zst";
-    sha256 = "sha256-Zb5m5TLJ1vcPKypZ3NliaL9oluz97ukTVGlOehuzyPU=";
+    hash = "sha256-sb06yszstKalc+b9rSuStRuY3YRebAL1Q4jEJkbGiMI=";
+    nativeBuildInputs = [ zstd ];
+    stripRoot = false;
   };
 
-  nativeBuildInputs = [ zstd ];
-
   buildCommand = ''
-    tar -xaf ${geoip}
-    tar -xaf ${extra}
-    mkdir -p $out/share
-    mv usr/share/GeoIP $out/share
+    mkdir -p $out/share/GeoIP
+    cp ${geoip}/usr/share/GeoIP/*.dat $out/share/GeoIP
+    cp ${extra}/usr/share/GeoIP/*.dat $out/share/GeoIP
   '';
 
   meta = with lib; {
```

Comment:
This `with lib;` is now considered bad practice iirc, can you remove it and just write `lib.` where it's necessary